jsonnet/prometheus-operator/alertmanagerconfigs-crd.json

```diff
@@ -236,28 +236,6 @@
                           "items": {
                             "description": "DiscordConfig configures notifications via Discord. See https://prometheus.io/docs/alerting/latest/configuration/#discord_config",
                             "properties": {
-                              "apiURL": {
```

Comment:
Why are we removing this?

Reply:
change`apiURL` to `webhook_url `  in new alerrmanager discord config.

ref: https://prometheus.io/docs/alerting/latest/configuration/#discord_config

Reply:
We can't remove them though, it would break users using old versions of alertmanager who still use this field.

I'd suggest to keep them but update the description mentioning change made upstream

Reply:
Got it!  Keep `apiURL` use this field.